### PR TITLE
Updated marabunta to fix allow_serie

### PR DESCRIPTION
**Pull request by tschanzt**

### CODE REVIEW

**Comment by leemannd** [comment]
LGTM unless travis failing

**Comment by tschanzt** [comment]
@leemannd updated to use 0.10.4. should fix the tests

File: HISTORY.rst [hunk 1]
```diff
@@ -28,6 +28,8 @@ Unreleased
 
 **Libraries**
 
+* Update marabunta to have fixed marabunta_serie
+
```
Comment (sebalix) [inline HISTORY.rst]: Can you also announce the breaking changes? Like in https://github.com/camptocamp/docker-odoo-project/pull/102/files#diff-d9ae695c8119e58c2ec435b6deae2d53

**Comment by tschanzt** [comment]
@sebalix updated.

**Comment by sebalix** [comment]
@tschanzt please rebase :)

**Comment by tschanzt** [comment]
@sebalix updated.

**Pull request merged**